right_image_topic: /wam_v/right_camera/right_image_raw
left_image_topic: /wam_v/left_camera/left_image_raw
publish_disparity: true